### PR TITLE
get-vim on windows

### PR DESCRIPTION
**Pull request by leinadcoder**
I added an option for tab/untab in visual mode and I made changes on the script to detect when running on Windows

### CODE REVIEW

**Comment by soonick** [comment]
@leinadcoder , can you create a different pull request for the tab/untab change, since it's not related to windows

File: README.md [hunk 1]
```diff
@@ -19,3 +19,18 @@ To download, compile and configure Vim:
...
+- [Cmdr](https://cmder.net/)
+- [Chocolatey](https://chocolatey.org/)
+
+To download, compile and configure Vim:
+
+```
+sh ./do-win.sh
+```
+
+**Note:** Run with administrator privilege as would with the sudo command.
```
Comment (soonick) [inline README.md:36]:
```suggestion
**Note:** Run with administrator privilege
```

File: README.md [hunk 2]
```diff
@@ -19,3 +19,18 @@ To download, compile and configure Vim:
...
+
+## Dependencies
+
+- [Cmdr](https://cmder.net/)
+- [Chocolatey](https://chocolatey.org/)
+
+To download, compile and configure Vim:
+
+```
+sh ./do-win.sh
```
Comment (soonick) [inline README.md:33]:
```suggestion
sh ./do.sh
```

File: do.sh [hunk 3]
```diff
@@ -200,6 +211,12 @@ map <down> <nop>
 map <left> <nop>
 map <right> <nop>
 
+" Change move between windows on cmder "
```
Comment (soonick) [inline do.sh:214]: What does this do?

File: do.sh [hunk 4]
```diff
@@ -1,36 +1,43 @@
...
-  echo ""
-  exit 1;
-fi
-
-sudo make install
+echo "install vim"
+
+case "$OSTYPE" in
+  msys*)
+    choco install vim;;
```
Comment (soonick) [inline do.sh:7]:
```suggestion
    choco install vim
    ;;
```

**Review by soonick — CHANGES_REQUESTED** [review]

**Comment by soonick** [comment]
Looks mostly good. Just some small things

File: do.sh (hunk 3 above)
Comment (leinadcoder) [reply do.sh:214]: When you use ctrl-w short cut, was dispatch close event on cmdr, I set new short cuts for a move between NERDTRee window and editor window.

**Comment by leinadcoder** [comment]
Work on suggested changes and create new pull requests

**Pull request closed without merging**